bin/martinize2

```diff
@@ -209,9 +222,26 @@ def entry():
     )
     known_mappings = read_mapping_directory(Path(DATA_PATH) / 'mappings')
 
-    from_ff = 'universal'
+    # Add user force fields and mappings
+    for directory in args.extra_ff_dir:
+        if not directory.is_dir():
+            raise ValueError('"{}" given to the -ff-dir option should be a directory.')
+        vermouth.forcefield.find_force_fields(directory, known_force_fields)
```

Comment:
Throw low, catch high. I'd rather put a try/except around the call to find_force_fields, and reraise it with a nicer error message than testing whether it's a directory beforehand.

Reply:
Done.